src/api/user.js

```diff
@@ -0,0 +1,7 @@
+import { client } from "../utils/httpClient";
+
+export const loginUser = (email) => client.get(`/users`, {
+  params: { email },
+});
```

Comment:
The `loginUser` function is using a GET request to retrieve user data based on email. Typically, login operations involve sending credentials via a POST request to authenticate the user. Consider using a POST request for login operations.